package.json

```diff
@@ -8,6 +8,9 @@
     "name": "jQuery Foundation and other contributors",
     "url": "https://github.com/jquery/jquery-color/blob/master/AUTHORS.txt"
   },
+  "scripts": {
+    "ci": "grunt"
```

Comment:
I'd prefer `grunt ci` and registering that task

On Wed, Feb 3, 2016 at 5:46 PM, Leo Balter <notifications@github.com> wrote:

> In package.json
> <https://github.com/jquery/jquery-color/pull/91#discussion_r51800889>:
>
> > @@ -8,6 +8,9 @@
> >      "name": "jQuery Foundation and other contributors",
> >      "url": "https://github.com/jquery/jquery-color/blob/master/AUTHORS.txt"
> >    },
> > +  "scripts": {
> > +    "ci": "grunt"
>
> grunt default task is defined as:
>
> grunt.registerTask( "default", [ "jshint", "qunit", "build", "compare_size" ] );
>
> maybe we could just call grunt jshint qunit here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jquery/jquery-color/pull/91/files#r51800889>.
>